spec/models/piece_spec.rb

```diff
@@ -225,44 +225,24 @@
       expect(king.y_position).to eq 3
     end
 
-    # it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
-    #   game = Game.create(white_user_id: 0, black_user_id: 1)
-    #   king = King.create(x_position: 4, y_position: 2, game_id: game.id)
-    #   rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id)
-    #   king.move_to!(4,3)
-    #   expect(king.x_position).to eq 4
-    #   expect(king.y_position).to eq 3
-    #   expect(rook.captured).to eq true
-    # end
+  #   it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
```

Comment:
@kyidkim i left a similar note on your other PR but i think it would make more sense to totally delete this test rather than push a commented out request.